Lib/importlib/util.py

```diff
@@ -235,6 +253,8 @@ def create_module(self, spec):
 
     def exec_module(self, module):
         """Make the module load lazily."""
+        import threading
```

Comment:
Why the inline import? We typically only do this if the import may fail, but I think every platform lets importing `threading` at least succeed.

Reply:
<details><summary><code>importlib.util</code> is an obligatory input, while <code>threading</code> is not:</summary>

```console
❯ python -X importtime -c 'import threading' 
import time: self [us] | cumulative | imported package
import time:       200 |        200 |   _io
import time:        41 |         41 |   marshal
import time:       460 |        460 |   posix
import time:       772 |       1471 | _frozen_importlib_external
import time:       112 |        112 |   time
import time:       248 |        360 | zipimport
import time:       157 |        157 |     _codecs
import time:       527 |        684 |   codecs
import time:       479 |        479 |   encodings.aliases
import time:       574 |       1736 | encodings
import time:       236 |        236 | encodings.utf_8
import time:        82 |         82 | _signal
import time:        40 |         40 |     _abc
import time:       203 |        242 |   abc
import time:       223 |        465 | io
import time:        49 |         49 |       _stat
import time:       234 |        282 |     stat
import time:       855 |        855 |     _collections_abc
import time:       101 |        101 |       genericpath
import time:       160 |        261 |     posixpath
import time:       517 |       1913 |   os
import time:       115 |        115 |   _sitebuiltins
import time:       448 |        448 |   _distutils_hack
import time:       178 |        178 |   types
import time:       137 |        137 |       warnings
import time:        89 |        226 |     importlib
import time:        66 |         66 |     importlib._abc
import time:        57 |         57 |         itertools
import time:        56 |         56 |         keyword
import time:        43 |         43 |           _operator
import time:       426 |        468 |         operator
import time:        94 |         94 |         reprlib
import time:        33 |         33 |         _collections
import time:       393 |       1099 |       collections
import time:        76 |         76 |         _functools
import time:       271 |        347 |       functools
import time:       246 |       1690 |     contextlib
import time:       142 |       2122 |   importlib.util
import time:        73 |         73 |   importlib.machinery
import time:        85 |         85 |   sitecustomize
import time:        36 |         36 |   usercustomize
import time:      1460 |       6428 | site
import time:       167 |        167 |   _weakrefset
import time:       387 |        553 | threading
```

</details> 

I assumed that adding `threading` would be unwelcome unless its functionality were actually needed. I can move the import to the top level.

Since `importlib.util` is imported by default while `threading` is not, I assumed that an extra obligatory input would be unwelcome unless thread-unsafe